src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -222,19 +205,34 @@ export class ExposureNotificationService {
     return false;
   }
 
+  private async *keysSinceLastFetch(
```

Comment:
I move this function next to `performExposureStatusUpdate`, so that it is easier to follow.